docs/source/index.rst

```diff
@@ -1,5 +1,5 @@
 ㊙️ python-barcode's documentation
-=================================
+===================================
```

Comment:
Two extra characters here?

Reply:
I get the following error (Py 2.7)
```
index.rst:2:Title underline too short.

\u3299\ufe0f python-barcode's documentation
=================================
```

and (Py 3.7):

```
index.rst:2:Title underline too short.

㊙️ python-barcode's documentation
=================================
```

Reply:
Huh, odd. I missed that warning before. Makes sense then.